content/guides/concepts/cid.md

```diff
@@ -5,18 +5,39 @@ menu:
         parent: concepts
 ---
 
-A *content identifier* is a value that addresses a single piece of content in IPFS. It is mainly a cryptographic hash of the content, but is encoded as a [multihash](https://github.com/multiformats/multihash) and [multicodec](https://github.com/multiformats/multicodec). (Note: older CIDs have a different design — see [version 0](#version-0) below.)
+## Summary
 
-<!-- TODO: explain more of the details of how CID v1 is composed here. -->
+A *content identifier*, or CID, is a label used for addressing content in IPFS. CID's are used as a standard way of pointing to pieces of information.  CID's identify specific pieces of content stored in IPFS.
 
-You can read up on the details in the [CID spec](https://github.com/ipld/cid). You might also want to check out the [CID inspector](http://cid-utils.ipfs.team/#zb2rhiVd5G2DSpnbYtty8NhYHeDvNkPxjSqA7YbDPuhdihj9L) for an interactive breakdown of CIDs.
+CID's are based on that content's cryptographic hash - a different piece of content will have a different hash and will produce different CID's.
 
-## Version 1
+## Format of CID's
+
+CID's can take a few different forms, each easy for humans and/or software to decode.  Any specific CID can be transformed to other equivalent CID representations (for example, using different base, CID version or codec).  
```

Comment:
Please don’t say “easy.” People in this community have come from a variety of backgrounds and expertise, and what’s easy for some is not so for others. When you use this word, you risk discouraging someone who’s perfectly smart and capable, but had a hard time trying to do this task because they’d never done it before.

> Any specific CID can be transformed to other equivalent CID representations

I don’t think we should say this. While you can up-convert a v0 CID to a v1 CID, there’s no explicit guarantee going forward that something similar will necessarily be possible if we ever invent a v2. Also, if you consider the hash as part of the CID, you cannot transform that (e.g. you couldn’t transform a SHA2-based CID to a SHA3-based CID if you all you have is the CID.)